prep-javascript-variables/index.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Variables</title>
+</head>
+<body>
+  <script>
+    var fullName="Yusuf Adanur";
+    var numberOfPets=0;
+    var likesVegetables=true;
+    var nothing=null;
+    var never;
+    console.log("My Name is "+ fullName);
+    console.log("Number of pets = "+numberOfPets);
+    console.log("Do I like vgetables? --> "+likesVegetables);
+    console.log("nothing is "+ nothing);
+    console.log("never is "+ never);
```

Comment:
Try putting a `,` instead of a `+` between your arguments for `console.log()`. This can become an issue when inspecting objects.

```js
console.log('My Name is', fullName);
```

Reply:
![Screen Shot 2021-02-23 at 8 15 53 PM](https://user-images.githubusercontent.com/11065204/108946763-f3b34380-7613-11eb-824d-0a80b3d1541a.png)